DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -233,6 +233,11 @@ void Executor::operator()(const Model::DispatchCommand& command)
     {
         Timer loopTimer, iterationTimer, bindTimer, dispatchTimer;
 
+        if (m_commandLineArgs.GetSetStablePowerState())
+        {
+            THROW_IF_FAILED(m_device->D3D()->SetStablePowerState(TRUE));
```

Comment:
Instead of throwing, can you log an warning or error that dev mode needs to be enabled?